matrix_client/api.py

```diff
@@ -582,6 +582,8 @@ def _send(self, method, path, content=None, query_params={}, headers={},
 
         if headers["Content-Type"] == "application/json" and content is not None:
             content = json.dumps(content)
+            
+        request_timeout = 10.0 + query_params.get("timeout", 30000) / 1000
```

Comment:
Any particular reason we're forcing this to be a float instead of an int?

Reply:
no. It is just because the requests_timeout could be a float. I will change it.